src/components/Messaging/container.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { PropTypes as PT } from 'prop-types';
+import { getEventMessages, sendMessage } from '../../services/holidayService';
+
+const Container = Wrapped =>
+  class extends React.Component {
+    static propTypes = {
+      toggleMessagingView: PT.func,
+      eventId: PT.number.isRequired,
+    };
+
+    constructor(props) {
+      super(props);
+      this.state = {
+        currentMessage: '',
+        messages: [],
+      };
+    }
+
+    componentDidMount() {
+      this.getMessages();
+    }
+
+    getMessages = () => {
+      getEventMessages(this.props.eventId).then(response => {
+        this.setState({ messages: response.data });
+      });
+    };
+
+    sendMessage = () => {
```

Comment:
Maybe rename this. I thought it was being called twice, but the other one is the service call.